crates/sui-rpc-api/proto/sui/rpc/v2alpha/archival_service.proto

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+import "google/protobuf/field_mask.proto";
+import "sui/rpc/v2alpha/checkpoint.proto";
+import "sui/rpc/v2alpha/executed_transaction.proto";
+import "sui/rpc/v2alpha/object.proto";
+
+// Alternitive names:
+// - HistoryService
+// - LedgerService
+// - HistoricalService
+// - HistoricalDataService
```

Comment:
Of these I like `HistoricalDataService` best, because then you can have a `LiveDataService` too.